cec.cpp

```diff
@@ -259,6 +262,19 @@ static PyObject * init(PyObject * self, PyObject * args) {
    return result;
 }
 
+static PyObject * close(PyObject * self, PyObject * args) {
+
+   if( PyArg_ParseTuple(args, "") == 0 && CEC_adapter == NULL ) {
```

Comment:
Since this function takes no arguments, you don't need to parse `args`.

I think the condition here should be `CEC_adapter != NULL`

Reply:
yep you're right, whoops